pkg/charts/cni/values.go

```diff
@@ -54,6 +54,22 @@ func NewValues() (*Values, error) {
 	return v, nil
 }
 
+// ToMap converts the Values intro a map[string]interface{}
+func (v *Values) ToMap() (map[string]interface{}, error) {
+	var valuesMap map[string]interface{}
+	rawValues, err := yaml.Marshal(v)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to marshal the values struct: %w", err)
+	}
+
+	err = yaml.Unmarshal(rawValues, &valuesMap)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to Unmarshal Values into a map: %w", err)
```

Comment:
I don't think the type we want to unmarshal into is very relevant for the error message. If I encounter this error, I'd want to know probably _what I'm trying to unmarshal_ so I know where the issue is. Perhaps we can re-word it a little?